### ISSUE TITLE
Support light/dark mode settings in UI

### ISSUE THREAD

**Issue by cweagans** [soflyy/oxygen-bugs-and-features#2288]
**Describe the feature you'd like to see included in Oxygen.**
Light mode/dark mode is available through a media query. Anywhere that I can set a color in the UI, I'd like to be able to override the color if a user is viewing the site in dark mode.

**Comment by KittenCodes** [comment]
Duplicate: https://github.com/soflyy/oxygen-bugs-and-features/issues/2269

**Comment by cweagans** [comment]
Fwiw, I don’t want easy dark mode. I want manual dark mode where I can specify the color overrides.

**Comment by pedrostein** [comment]
@cweagans this would be a real great feature to have in oxygen. I tried a few Dark Mode plugins, but without being very happy with them.

One of the best solutions I came across was by using Microthemer where one can add a media query for dark mode.

Also there is a tutorial for oxygen on youtube https://www.youtube.com/watch?v=CzfP3-xwoiM which sort of is ok, but its more focused about adding the switching button. Though as you mentioned, for editing dark there is a bit more required.

Personally I think  think it would be most awesome and user friendly if we would have a a button in the editor similar to the breakpoints where we can change to dark mode with preview and make our edits the way we see fit. 
Like this we could also easy edit dark mode in the different break points. 
Also having then a module for the light- / dark mode switch for use in the front-end would be very welcome.
